crates/stack/src/device.rs

```diff
@@ -100,32 +99,39 @@ impl CaptureDevice {
     }
 }
 
-impl<'a> phy::Device<'a> for CaptureDevice {
-    type RxToken = RxToken<'a>;
-    type TxToken = TxToken<'a>;
+impl phy::Device for CaptureDevice {
+    type RxToken<'a> = RxToken<'a>
+    where
+        Self: 'a;
+    type TxToken<'a> = TxToken<'a>
+    where
+        Self: 'a;
 
-    fn receive(&'a mut self) -> Option<(Self::RxToken, Self::TxToken)> {
+    fn receive(&mut self, timestamp: Instant) -> Option<(Self::RxToken<'_>, Self::TxToken<'_>)> {
```

Comment:
In 0.10 `timestamp` arrives in `fn receive`, so I just store in `Tx/RxToken` and use it in `fn consume`.
In 0.8.1 `timestamp` was arriving in `fn consume`.
No idea🤷‍♂️
Works.